cmd/habitat/data_proxy/data_proxy.go

```diff
@@ -31,6 +31,10 @@ type ReadRequest struct {
 	Body        json.RawMessage `json:"body"`
 }
 
+type ReadResponse struct {
```

Comment:
Since we are adding to the Habitat API, can we put these rrequest and response structs in `ctl/message.go`? This prevents `habitatctl` from having any dependency on any of the `habitat packages

Reply:
Also, add these types to the `requestType` switching function. I know its gross, but will be needed until we switch to using a better API library like `echo` or something